libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -255,7 +258,7 @@ void AP_ADSB::update_vehicle(const mavlink_message_t* packet)
         if (!is_zero(_lowest_threat_distance) && // nonzero means it is valid
             _ahrs.get_position(my_loc)) {       // true means my_loc is valid
 
-            float distance = get_distance(my_loc, get_location(vehicle));
+            float distance = get_distance(my_loc, vehicle_location);
```

Comment:
why not just call vehicle.get_location() here? instead of grabbing it at the beginning of the function?